degov_search_content/config/update_8008/install/views.view.search_content.yml

```diff
@@ -0,0 +1,251 @@
+langcode: de
```

Comment:
Is there a need to include the whole yml file here? Are there any other changes than min_length in the update? Please use the rewrite folder to change the min_length.